loops.js

```diff
@@ -17,9 +20,15 @@
 // loop...
 // klaar!
  // ==========================================
-
-
-
+for (let i=0; i < 5; i++) {
```

Comment:
Hier hetzelfde als bij de 3x hoera! opdracht, de let i hoeft niet perse maar is ook niet fout